src/components/modals/WalletListModal.tsx

```diff
@@ -136,11 +145,11 @@ export function WalletListModal(props: Props) {
   const handlePaymentMethodPress = useHandler((paymentMethodId: string) => () => {
     bridge.resolve({ wyreAccountId: paymentMethodId })
   })
-  const handleWalletListPress = useHandler((walletId: string, currencyCode: string) => {
+  const handleWalletListPress = useHandler((walletId: string, currencyCode: string, tokenId?: string, customAsset?: CustomAsset) => {
```

Comment:
Why was tokenId added here?

Reply:
It would complain where referenced after adding customAsset